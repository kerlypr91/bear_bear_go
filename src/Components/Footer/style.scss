.footer {
  position: absolute;
  padding: 16px;
  bottom: 0;

  &__copy {
    color: white;
  }
}
