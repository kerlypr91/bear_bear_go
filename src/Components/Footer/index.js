import React from 'react'

import './style.scss'


export default function Footer (props) {
  return (
    <footer className="footer">
      <span className="footer__copy">&copy; BCC - Bear Code Company</span>
    </footer>
  )
}