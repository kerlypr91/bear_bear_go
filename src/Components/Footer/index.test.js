import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index';

test('Footer match snapshot', () => {
  const component = renderer.create(
    <Footer />
  )

  // base snapshot

  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()
});