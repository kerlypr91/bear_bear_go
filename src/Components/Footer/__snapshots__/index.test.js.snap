// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer match snapshot 1`] = `
<footer
  className="footer"
>
  <span
    className="footer__copy"
  >
    Â© BCC - Bear Code Company
  </span>
</footer>
`;
